portal/src/graphql/adminapi/GroupsList.module.css

```diff
@@ -0,0 +1,48 @@
+.root {
+  display: flex;
+  flex-direction: column;
+}
+
+.listWrapper {
+  flex: 1 0 0%;
+  min-height: 0;
+  overflow-y: auto;
+}
+
+.pagination {
+  align-self: flex-end;
+  margin: 12px 16px;
+}
+
+.actionButton {
+  padding-left: 0;
+  border: 0;
+}
+
+.actionButtonText {
+  font-weight: 600;
+  font-size: 12px;
+}
+
+.cell {
+  display: flex;
+  width: 100%;
+  height: 100%;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.cellText {
+  @apply truncate;
+  font-size: 14px;
+  line-height: 20px;
+}
+
+.description {
+  white-space: normal;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  font-size: 12px;
+  line-height: 16px;
+}
```

Comment:
All lists in roles and groups should share same styles, maybe we should create common components to contain these styles.

We can do it in separated issue.

Reply:
opened https://github.com/authgear/authgear-server/issues/3967